sdk/storage/Azure.Storage.Blobs/api/Azure.Storage.Blobs.netstandard2.0.cs

```diff
@@ -1417,6 +1438,27 @@ public partial class BlobBaseClient
         protected virtual Azure.Storage.Blobs.Specialized.BlobBaseClient WithSnapshotCore(string snapshot) { throw null; }
         public virtual Azure.Storage.Blobs.Specialized.BlobBaseClient WithVersion(string versionId) { throw null; }
     }
+    public partial class BlobDirectoryClient
```

Comment:
I wonder if this should go to `Specialized` namespace. We can keep this as open question.

Reply:
I think leaving it in the `Specialized` namespace makes sense, considering we also put `BlockBlobClient`, `AppendBlobClient`, and `PageBlobClient` in the specialized namespace.